appmap/_implementation/recording.py

```diff
@@ -229,7 +234,8 @@ def wrapped_exec_module(*args, **kwargs):
                      args,
                      kwargs)
         exec_module(*args, **kwargs)
-        Recorder().do_import(*args, **kwargs)
+        import_fn(*args, **kwargs)
```

Comment:
Is passing this as argument solely for testing? If so, could we monkey patch instead?

Reply:
Yes, it's solely for testing. In my haste to get this finished, I missed `monkeypatch.setattr`, and the alternatives were all more complicated than I wanted.

I changed it to use `monkeypatch.setattr`.